systems/common/msgbus/qpub.go

```diff
@@ -55,6 +57,27 @@ func (q *qPub) Publish(payload any, routingKey string) error {
 	return nil
 }
 
+func (q *qPub) PublishProto(payload proto.Message, routingKey string) error {
+
+	b, err := proto.Marshal(payload)
+	if err != nil {
+		return err
```

Comment:
Can you use logrus to log the errors when marshal does not and when function fail to publish.I think it will useful in debugging.